content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,283 @@
+
+### Enable extra Agent checks
+
+For each extra Agent check you want to enable across your deployment, add its configuration under the `properties.dd.integrations` key, e.g.:
+
+~~~
+  properties:
+    dd:
+      integrations:
+        directory:
+          init_config: {}
+          instances:
+            directory: "."</code></pre>
+        #process:
+        #  init_config: {}
+        #...
+~~~
+
+The configuration under each check name should look the same as if you were configuring the check in its own file in the Agent's conf.d directory.
+
+You cannot configure a check for a subset of nodes in your deployment; everything you configure in `runtime.yml` will apply to every node.
+
+To customize configuration for the default checks—system, network, disk, and ntp—see the [full list of configuration options](https://github.com/DataDog/datadog-agent-boshrelease/blob/master/jobs/dd-agent/spec) for the Datadog Agent BOSH release.
+
+### Sync the runtime configuration to the Director
+
+~~~
+# BOSH CLI v1
+bosh update runtime-config runtime.yml
+
+# BOSH CLI v2
+bosh update-runtime-config runtime.yml
+~~~
+
+### Redeploy your Cloud Foundry deployment
+
+~~~
+# BOSH CLI v1
+bosh deployment yourDeploymentManifest.yml
+bosh -n deploy
+
+# BOSH CLI v2
+bosh -n -d yourDeployment deploy yourDeploymentManifest.yml
+~~~
+
+Since runtime configuration applies globally, BOSH will redeploy every node in your deployment. If you have more than one deployment, redeploy those, too, to install the Datadog Agent everywhere.
+
+### Verify the Agent is installed everywhere
+
+The easiest way to check that Agent installs were successful is to filter for them in the [Host map page](https://app.datadoghq.com/infrastructure/map) in Datadog. The Agent BOSH release tags each host with a generic `cloudfoundry` tag, so filter by that, and optionally group hosts by any tag you wish (e.g. `bosh_job`), as in the following screenshot:
+
+![cloud-foundry-host-map](/static/images/cloud-foundry-host-map.png)
+
+Click on any host to zoom in, then click 'system' within its hexagon to make sure Datadog is receiving metrics for it:
+
+![cloud-foundry-host-map-detail](/static/images/cloud-foundry-host-map-detail.png)
+
+# Deploy the Datadog Firehose Nozzle
+
+As with the Datadog Agent, Datadog provides a BOSH release of the Datadog Firehose Nozzle. After uploading the release to your Director, you can add the Nozzle to an existing deployment, or create a new deployment that only includes the Nozzle. The instructions below assume you're adding it to an existing Cloud Foundry deployment that has a working Loggregator Firehose.
+
+### Upload Datadog's release to your BOSH Director
+
+~~~
+# BOSH CLI v1
+bosh upload release https://github.com/DataDog/datadog-firehose-nozzle-release/releases/download/61/datadog-firehose-nozzle-release-61.tgz
+
+# BOSH CLI v2
```

Comment:
There will be one once we get it uploaded to s3. I'm working on that. it'll be something like:

https://cloudfoundry.datadoghq.com/datadog-firehose-nozzle/datadog-firehose-nozzle-release-latest.tgz